src/main/java/DoneCommand.java

```diff
@@ -0,0 +1,14 @@
+class DoneCommand extends UserCommand {
+    private int index;
+
+    public DoneCommand (int index, TaskList tasks) {
+        super(tasks);
+        this.index = index;
```

Comment:
Maybe you could use a name which is more obvious such as taskId or taskNumber instead of index?